GliaWidgets/Sources/Component/AttachmentList/Item/AttachmentSourceItemStyle.swift

```diff
@@ -36,7 +35,7 @@ public class AttachmentSourceItemStyle: Equatable {
     ///   - icon: Icon of the item. Default is one of three icons (Take Photo, Photo Library or Browse) corresponding to the kind of attachment.
     ///   - iconColor: Color of the icon.
     ///   - accessibility: Accessibility related properties.
-    ///
+    ///   
```

Comment:
Is space required here?

Reply:
fixed [d2cf4a7](https://github.com/salemove/ios-sdk-widgets/pull/887/commits/d2cf4a7c8c63bcb2a3064f0ed6f678ec4995ac7f)